src/MSBuildLocator/DotNetSdkLocationHelper.cs

```diff
@@ -112,7 +123,33 @@ private static string GetDotNetBasePath(string workingDirectory)
                 return null;
             }
 
-            return matched.Groups[1].Value.Trim();
+            var lineSdkIndex = lines.FindIndex(line => line.Contains(".NET Core SDKs installed") || line.Contains(".NET SDKs installed"));
```

Comment:
Suggestion: Run `dotnet --list-sdks` instead of `dotnet --info` and you wouldn't have to match on complex human readable strings.

Reply:
Fixed. It now uses --list-sdks.